mkdocs.yml

```diff
@@ -1,36 +1,46 @@
 site_name: Autolab Documentation
 site_url: "https://docs.autolabproject.com/"
+extra_css:
+    - stylesheets/extra.css
 nav:
     - Autolab:
-          - Getting Started: index.md
-          - Guide for Instructors: instructors.md
-          - Guide for Lab Authors: lab.md
-          - Features:
+        - Getting Started: index.md
+        - Guide for Instructors: instructors.md
+        - Guide for Lab Authors: lab.md
+        - Features:
             - Formatted Feedback: features/formatted-feedback.md
             - Scoreboards: features/scoreboards.md
             - Embedded Forms: features/embedded-forms.md
             - Annotations: features/annotations.md
             - MOSS Plagiarism Detection: features/moss.md
             - Metrics: features/metrics.md
+    - Installation:
+        - Installation Overview: installation/overview.md
+        - Docker Compose Install: installation/docker-compose.md
+        - Autolab:
+            - Ubuntu 18.04: installation/ubuntu.md
+            - Mac OSX 10.11+: installation/osx.md
+        - Tango: installation/tango.md
+        # - (Legacy) OneClick Install: installation/one-click.md
```

Comment:
somehow the legacy one click install is still appearing. Could you check this?

Reply:
resolved, not replicable